pkg/operator/ceph/object/bucket/s3-handlers.go

```diff
@@ -0,0 +1,243 @@
+/*
```

Comment:
Should we try to leverage this code with the code in the aws s3 provisioner into a common pkg? I think there is a fair amount of duplication but I like the approach here better.

Reply:
Agree. Deferred for now due to time constraints.